src/main/webapp/pipe.js

```diff
@@ -371,6 +374,32 @@ function triggerRebuild(taskId, project, buildId) {
     });
 }
 
+function triggerBuild(taskId) {
+    Q("#rebuild-" + taskId).hide();
+
+    var before;
+    if (crumb.value != null && crumb.value != "") {
+        console.info("Crumb found and will be added to request header");
+        before = function(xhr){xhr.setRequestHeader(crumb.fieldName, crumb.value);}
+    } else {
+        console.info("Crumb not needed");
+        before = function(xhr){}
+    }
+
+    Q.ajax({
+        url: rootURL + "/job/" + taskId + '/build?delay=0sec',
```

Comment:
We must require tests for all pull requests to be able to regression test. 

Marcus Philip
+46 70 767 46 77

> 4 apr 2015 kl. 20:15 skrev Patrik Boström <notifications@github.com>:
> 
> In src/main/webapp/pipe.js:
> 
> > @@ -371,6 +374,32 @@ function triggerRebuild(taskId, project, buildId) {
> >      });
> >  }
> >  
> > +function triggerBuild(taskId) {
> > +    Q("#rebuild-" + taskId).hide();
> > +
> > +    var before;
> > +    if (crumb.value != null && crumb.value != "") {
> > +        console.info("Crumb found and will be added to request header");
> > +        before = function(xhr){xhr.setRequestHeader(crumb.fieldName, crumb.value);}
> > +    } else {
> > +        console.info("Crumb not needed");
> > +        before = function(xhr){}
> > +    }
> > +
> > +    Q.ajax({
> > +        url: rootURL + "/job/" + taskId + '/build?delay=0sec',
> Not sure if this will work if the job is in a folder.
> 
> —
> Reply to this email directly or view it on GitHub.
>